tests/autoyast/prepare_profile.pm

```diff
@@ -53,6 +53,9 @@ sub run {
         next unless my ($value) = get_var($var);
         $profile =~ s/\{\{$var\}\}/$value/g;
     }
+    if (check_var('IPXE', '1')) {
+        $path = get_required_var('SUT_IP') + $path;
```

Comment:
Just a question. Where is this SUT_IP given? I see it in vars.json but not in settings tab.

Reply:
This comes from the machine configuration through the backend.

Reply:
ok, thanks for the info.